### DISCUSSION TITLE
I can't believe GitHub blocked my repository

### DISCUSSION THREAD

**Discussion by Mabbs** [community/community#23593] [New to GitHub]
<p>I can’t understand what my repository is violation GitHub’s TOS, this is just a blog.<br>
What’s more incredible is that GitHub directly banned my repository without any notice, I have submitted the ticket, but so far the GitHub staff has not given me a response.<br>
Maybe I should try other products.<br>
My repository URL: <a href="https://github.com/Mabbs/mabbs.github.io" rel="noopener nofollow ugc">https://github.com/Mabbs/mabbs.github.io</a><br>
Last updated backup file: <a href="https://mayx-my.sharepoint.com/:u:/g/personal/mayx_say-huahuo_com/EVfl7uzid0hLuXTGAemlFdIBnfdH0fbX6yBbwlG_WCv1tg?e=g8YA8W" rel="noopener nofollow ugc">https://mayx-my.sharepoint.com/:u:/g/personal/mayx_say-huahuo_com/EVfl7uzid0hLuXTGAemlFdIBnfdH0fbX6yBbwlG_WCv1tg?e=g8YA8W</a></p>

**Comment by jfullstackdev** [comment]
<p>what do you think is the possible reason why they banned the repo?</p>

**Comment by Mabbs** [comment]
<p>idk, I can’t understand.I don’t think my blog has anything to offend GitHub.</p>

**Comment by jfullstackdev** [comment]
<p>what is your content all about?</p>
<p>anyways, just try to reach out here if you are not yet trying</p>
<p><a href="https://support.github.com/contact/account">https://support.github.com/contact/account</a></p>

**Comment by Mabbs** [comment]
<p>Some articles about technology and my life.<br>
I have opened a ticket, but so far no employee has replied.</p>

**Comment by Mabbs** [comment, marked as answer]
<p>Problem has been solved.</p>

**Comment by matthewpersico** [comment]
<p>So, what happened? Why did they block it in the first place?</p>

**Comment by Mabbs** [comment]
<p>I made a web page that can redirect to any website and put it on GitHub pages. Because <a href="http://git.io" rel="noopener nofollow ugc">git.io</a> URL shortener service can be used for the address of GitHub pages. Someone discovered my service and directed it to a malicious site, so GitHub staff blocked me.</p>
